django_celery_results/backends/database.py

```diff
@@ -87,8 +87,8 @@ def _get_extended_properties(self, request, traceback):
             if task_kwargs is not None:
                 _, _, task_kwargs = self.encode_content(task_kwargs)
 
-            properties = getattr(request, 'properties', {}) or {}
-            periodic_task_name = properties.get('periodic_task_name', None)
+            periodic_task_name = getattr(request, 'periodic_task_name', None)
```

Comment:
thanks for the fix, curious, if removing properties create any regression or not? can you please elaborate the change bit more?

Reply:
This change reduces complexity by directly accessing the `periodic_task_name` attribute from request instead of an additional lookup inside properties. If `request.periodic_task_name` exists, it retrieves the value directly; otherwise, it defaults to `None`. The main reason for the change is that by passing the name in the headers of `apply_async()`, it is no longer available in properties, and is instead available in request. Removing properties shouldn't create any regression because it was only being used for the purpose of retrieving the `periodic_task_name`, and isn't being returned by the `_get_extended_properties` function or accessed in any other part of the code.